SPECS/openssh/openssh.spec

```diff
@@ -25,6 +26,10 @@ BuildRequires:  krb5-devel
 BuildRequires:  openssl-devel
 BuildRequires:  pam-devel
 BuildRequires:  systemd
+%if %{with_check}
+BuildRequires:  %{_sbindir}/useradd
```

Comment:
I'd go with `shadow-utils` instead. That's what we've been doing so far, it's good to be consistent.

Reply:
Fixed, thanks